modules/gapi/misc/python/pyopencv_gapi.hpp

```diff
@@ -366,11 +366,11 @@ static PyObject* pyopencv_cv_GOut(PyObject* , PyObject* py_args, PyObject* kw)
 
 static cv::detail::OpaqueRef extract_opaque_ref(PyObject* from, cv::detail::OpaqueKind kind)
 {
-#define HANDLE_CASE(T, O) case cv::detail::OpaqueKind::CV_##T:  \
-{                                                               \
-    O obj{};                                                    \
```

Comment:
Isn't it calling the default constructor already? What about defining it simply as `O obj;` ?